### DISCUSSION TITLE
[General]

### DISCUSSION THREAD

**Discussion by benmahria** [glpi-project/glpi-agent#697] [General]
### Your question

When I initiate a deploy on demand from the administration computer page I notices that it create a [deploy on demand] task configurated to that specific computer.

Once it finish the deploy the task is still there and the "Jobs configuration" it continue to list the "old" computers that asked for a deploy.

Is it normal?
Why GLPI Inventory doesn't delete the "actor" once the deploy is been done correctly?

**Comment by g-bougard** [comment]
Hi @benmahria 

please, set a title. Also, this question is not related to glpi-agent directly. You should probably ask your question on glpi forum, in the plugin section.
